core/lib/presentation/views/dialog/subaddressing_dialog_builder.dart

```diff
@@ -0,0 +1,330 @@
+
+import 'package:core/core.dart';
+import 'package:core/presentation/views/dialog/confirm_dialog_button.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+
+typedef OnConfirmButtonAction = void Function();
+typedef OnCancelButtonAction = void Function();
+typedef OnCloseButtonAction = void Function();
+typedef OnCopyButtonAction = void Function();
+
+class SubaddressingDialogBuilder {
```

Comment:
We already have such a dialog builder, why don't you reuse it?

Reply:
i want my popup to have two content "fields": some text ("You are about to..."), and below that, address display along with a `copy` button
![image](https://github.com/user-attachments/assets/3e322534-e227-48d4-b712-353d64bff43b)

i don't think the available dialog builders make something like this possible ?

Reply:
You can still do it from our existing DialogBuilder abstract class. You just need to create a parameter for it and add a condition to render that widget. IMO, it quite easy, please take the time to research it.